src/jit/compiler.h

```diff
@@ -8843,6 +8843,8 @@ void * CompAllocator::ArrayAlloc(size_t elems, size_t elemSize)
 inline
 LclVarDsc::LclVarDsc(Compiler* comp)
     :
+#ifndef _TARGET_X86_
+    // TODO-x86 - Need initialization of the _lvArgReg and _lvOtherArgReg for x86?
```

Comment:
This doesn't make sense to me. Why does initializing them to REG_STK cause a problem?